Model/ResourceModel/CustomerCreditCard/Collection.php

```diff
@@ -53,4 +53,26 @@ public function getCreditCards($customerId, $boltConsumerId, $boltCreditCardId){
     public function doesCardExist($customerId, $boltConsumerId, $boltCreditCardId){
         return $this->getCreditCards($customerId, $boltConsumerId, $boltCreditCardId)->getSize() > 0;
     }
+
+    /**
+     * @param $customerId
+     * @return array
+     */
+    public function getCreditCardInfosByCustomerId($customerId)
```

Comment:
@johnmccombs1  In which PR do you use this method?

Reply:
@YevhenShBolt It’s called in this PR. See here https://github.com/BoltApp/bolt-magento2/pull/443/files#diff-822f1bdec8cc9ace5f31aaba083f6b84R142